src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/StringItem.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\Field\FieldTypeExtend;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\Plugin\Field\FieldType\StringItem as StringItemExtend;
+
+/**
+ * {@inheritdoc}
+ */
+class StringItem extends StringItemExtend {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    switch ($field_definition->getName()) {
+      // Random values for orcid need to match the regex ^[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]$.
+      case 'field_person_orcid':
```

Comment:
where can a list of these fully qualified field names be found? Is there a table or an admin page that would list them or purely by convention?

Reply:
The fields each have their own database tables but you can find them in the admin UI of `journal-cms`. Either as a field against the content type (e.g. `/admin/structure/types/manage/annual_report/fields`) or as a field against a paragraph type (e.g. `/admin/structure/paragraphs_type/affiliation/fields`)